### PR TITLE
WIP : Updated text plus preference of singularity profile

### PR DESCRIPTION
**Pull request by chriswyatt1**
Update to UCL Myriad config

Made all export env lines universal with $HOME
Requested users use `-profile singularity`, as apptainer does not work for some pipelines. 

@FernandoDuarteF , could you double check with works please. Would be good to quickly check this all works for a couple of pipelines, but should be fine now. Thanks

### CODE REVIEW

**Comment by chriswyatt1** [comment]
I also think that maybe we should force singularity then to be used in the ucl_myriad config as done here:

https://github.com/nf-core/configs/blob/master/conf/eddie.config

As there is now other container engine recommended to work on myriad

File: docs/ucl_myriad.md [hunk 1]
```diff
@@ -43,3 +49,13 @@ Then make sure that your bin PATH is executable, by placing the following line i
 ```bash
 export PATH=$PATH:/home/<YOUR_ID>/bin
 ```
+
+## Running an nf-core pipeline
+
+To run the pipeline, make sure to add `-profile ucl_myriad,singularity`. This will download and launch the [`ucl_myriad.config`](../conf/ucl_myriad.config) which has been pre-configured with a setup suitable for the myriad cluster, as well as use the correct container engine for myriad.
```
Comment (jfy133) [inline docs/ucl_myriad.md:55]: Why not just set `singularity = true` in the config itself?

**Comment by jfy133** [comment]
Ah what you said @chriswyatt1 :sweat_smile:

**Comment by jfy133** [comment]
@nf-core-bot fix linting

File: docs/ucl_myriad.md (hunk 1 above)
Comment (chriswyatt1) [reply docs/ucl_myriad.md:55]: Great, we will do that :)

**Comment by FernandoDuarteF** [comment]
@nf-core-bot fix linting

**Comment by chriswyatt1** [comment]
@FernandoDuarteF - last commit added the CacheDir line. So should be ready now

**Review by FernandoDuarteF — APPROVED** [review]
Good to go

**Pull request merged**